tests/benchdnn/graph/deserialize.cpp

```diff
@@ -143,22 +143,22 @@ op deserialized_op_t::create() const {
         const auto &attr_value = it->second;
         const auto &type = attr_value.type_;
         if (type == "string") {
-            const auto value = attr_value.str_value_;
+            const auto &value = attr_value.str_value_;
             aop.set_attr(attr, value);
         } else if (type == "bool") {
-            const auto value = attr_value.bool_value_;
+            const auto &value = attr_value.bool_value_;
```

Comment:
just curious to know if coverity really complains for these POD ? seems POD by value is better than by reference.